packages/optimizer/lib/transformers/PreloadHeroImage.js

```diff
@@ -69,17 +69,9 @@ class PreloadHeroImage {
       preload.attribs.media = heroImage.media;
     }
     if (heroImage.srcset) {
```

Comment:
I wonder if this should be configurable? Should there be a flag to force inclusion of images that have `srcset`?

Reply:
That would cause double image downloads which is something I'd prefer to avoid.

Reply:
But only on non-Chromium browsers. It's not needed now, but it would be useful in the near future to evaluate the performance impact for pages that use `srcset`.